python/mxnet/initializer.py

```diff
@@ -265,3 +268,19 @@ def _init_weight(self, _, arr):
             random.normal(0, scale, out=arr)
         else:
             raise ValueError("Unknown random type")
+
+class MSRAPrelu(Xavier):
+    """Initialize the weight with initialization scheme from
+        Delving Deep into Rectifiers: Surpassing Human-Level Performance on ImageNet Classification.
+
+    Parameters
+    ----------
+    factor_type: str, optional
+        Use ```avg```, ```in```, or ```out``` to init
+
+    slope: float, optional
+        initial slope of any PReLU (or similar) nonlinearities.
+    """
+    def __init__(self, factor_type="avg", slope=0.25):
+        magnitude = 2. / (1 + slope ** 2)
+        super(MSRAPrelu, self)__.init__("gaussian", factor_type, magnitude)
```

Comment:
Please fix gramma error and rebase. We can merge this in as long as it doesn't break existing code